pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -2537,6 +2537,7 @@ default-package-overrides:
 
 extra-packages:
   - aeson < 0.8                         # newer versions don't work with GHC 7.6.x or earlier
+  - Agda == 2.6.1                       # Fix agda version
```

Comment:
Please, add an explanation into the code why it is a static version, so people after you would know the cause of the configuration.

Reply:
I've put "allows agdaPackages to depend on a fixed version"